ansible/projects/ovn-kubernetes/deploy-test-pods.yaml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Create ovn-kubernetes pod {{ pod_name }}
+  k8s:
+    state: present
+    wait: true
+    wait_timeout: 300
+    wait_condition:
+      type: Ready
+      status: "True"
+    kubeconfig: "{{ kubeconfig }}"
+    context: "{{ k8s_context }}"
+    resource_definition:
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: "{{ pod_name }}"
+        namespace: default
+        annotations:
+          k8s.ovn.org/pod-networks: |
+            {"default":{"ip_addresses":["192.168.0.8/24"],
+            "mac_address":"0a:58:c0:a8:00:08",
+            "gateway_ips":["192.168.0.1"],"ip_address":"192.168.0.8/24",
+            "gateway_ip":"192.168.0.1"}}
+        labels:
+          app: ovn-kubernetes
+      spec:
+        containers:
+          - name: "{{ pod_name }}"
+            image: "{{ pod_image }}"
+            command: ["/bin/sh", "-c", "sleep infinity"]
+- name: Get pod {{ pod_name }}
```

Comment:
maybe this also need to be in a global place

Reply:
Updated